tool/gocross/gocross-wrapper.sh

```diff
@@ -21,6 +21,15 @@ repo_root="$(dirname $0)/../.."
 
 toolchain="$HOME/.cache/tailscale-go"
 
+if [ -d "$toolchain" ]; then
+    # A toolchain exists, but is it recent enough to compile gocross? If not,
+    # wipe it out so that the next if block fetches a usable one.
+    wantGoMinor=$(egrep '^go ' "$repo_root/go.mod" | cut -f2 -d'.')
```

Comment:
I think `snake_case` is more typical for script variables, but YMMV.

Reply:
Oops, yeah. Done.